painttiming.bs

```diff
@@ -179,8 +188,11 @@ The {{PerformancePaintTiming}} interface {#sec-PerformancePaintTiming}
 * The {{PerformanceEntry/startTime}} attribute's getter must return a {{DOMHighResTimeStamp}} of when the paint occured.
 * The {{PerformanceEntry/duration}} attribute's getter must return 0.
 
-NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} for {{Window}} contexts.
-This allows developers to detect support for paint timing.
+Extension to {{PerformanceObserver/supportedEntryTypes}} {#sec-extendSupportedEntryTypesWithPaint}
```

Comment:
In all other performance specs, we went with a note in the spec itself, and a [registry](https://w3c.github.io/timing-entrytypes-registry/#registry) that contains the normative conditions. I wonder if we could add those normative conditions in the registry, to maintain consistency.

Reply:
Ah I see. So maybe use a non-normative note for the optional-per-context thing?